lib/queue.js

```diff
@@ -563,17 +563,15 @@ class Queue extends Emitter {
       throw new Error(`unable to update the status of ${status} job ${job.id}`);
     }
 
-    const multi = this.client
-      .multi()
-      .lrem(this.toKey('active'), 0, job.id)
-      .srem(this.toKey('stalling'), job.id);
-
     const jobEvent = {
       id: job.id,
       event: status,
       data: err ? err.message : data,
     };
 
+    let removeJob = false;
+    let delayedTime;
```

Comment:
could move the default here
```suggestion
    let delayedTime = '';
```

Reply:
I thought it was maybe clearer to have the default in the call.

Reply:
sure - I'm impartial